docs/data-sources/pipeline.md

```diff
@@ -0,0 +1,26 @@
+# Data Source: pipeline
+
+Use this data source to look up properties on a specific pipeline. This is
+particularly useful for looking up the webhook URL for each pipeline.
+
+Buildkite Documentation: https://buildkite.com/docs/pipelines
+
+## Example Usage
+
+```hcl
+data "buildkite_pipeline" "repo2" {
+    slug = "repo2"
+}
+```
+
+## Argument Reference
+
+* `slug` - (Required) The slug of the pipeline, available in the URL of the pipeline on buildkite.com
+
+## Attributes Reference
+
+* `description` - A description of the pipeline.
+* `default_branch` - The default branch to prefill when new builds are created or triggered.
```

Comment:
this is true, _and_ the default branch is also always displayed at the top in the pipeline UI. not sure if it's helpful to add it's usually the trunk/main or something?